### PR TITLE
[read-fonts] Improve markdown in `int_set`

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

File: read-fonts/src/collections/int_set/bitset.rs [hunk 1]
```diff
@@ -492,16 +494,16 @@ impl BitSet {
         Self::major_start(major) + (PAGE_BITS - 1)
     }
 
-    /// Returns the index in self.pages (if it exists) for the page with the same major as major_value.
+    /// Returns the index in `self.page_map` (if it exists) for the page with the same major as `major_value`.
```
Comment (garretrieger) [inline read-fonts/src/collections/int_set/bitset.rs:497]: Should be `self.pages`

File: read-fonts/src/collections/int_set/bitset.rs [hunk 2]
```diff
@@ -492,16 +494,16 @@ impl BitSet {
...
         self.page_map
             .binary_search_by(|probe| probe.major_value.cmp(&major_value))
             .ok()
             .map(|info_idx| self.page_map[info_idx].index as usize)
     }
 
-    /// Returns the index in self.pages for the page with the same major as major_value. Will create the page
-    /// if it does not yet exist.
+    /// Returns the index in `self.page_map` for the page with the same major as `major_value`. Will create
+    /// the page if it does not yet exist.
```
Comment (garretrieger) [inline read-fonts/src/collections/int_set/bitset.rs:506]: Should be `self.pages`

**Review by garretrieger — APPROVED** [review]
Thanks!

File: read-fonts/src/collections/int_set/bitset.rs (hunk 1 above)
Comment (waywardmonkeys) [reply read-fonts/src/collections/int_set/bitset.rs]: Whoops, this and the other fixed.

**Review by garretrieger — APPROVED** [review]

**Pull request merged**